alacritty/src/event.rs

```diff
@@ -286,23 +329,8 @@ impl<'a, N: Notify + 'a, T: EventListener> input::ActionContext<T> for ActionCon
         self.clipboard
     }
 
-    /// Spawn URL launcher when clicking on URLs.
-    fn launch_url(&self, url: Url) {
-        if self.mouse.block_url_launcher {
-            return;
-        }
-
-        if let Some(ref launcher) = self.config.ui_config.mouse.url.launcher {
-            let mut args = launcher.args().to_vec();
-            let start = self.terminal.visible_to_buffer(url.start());
-            let end = self.terminal.visible_to_buffer(url.end());
-            args.push(self.terminal.bounds_to_string(start, end));
-
-            match start_daemon(launcher.program(), &args) {
-                Ok(_) => debug!("Launched {} with args {:?}", launcher.program(), args),
-                Err(_) => warn!("Unable to launch {} with args {:?}", launcher.program(), args),
-            }
-        }
+    fn scheduler(&mut self) -> &mut Scheduler {
```

Comment:
Could you rename it to `scheduler_mut`? you can also update `clipboard` to `clipboard_mut` as well.

Reply:
I'm not sure if that's necessary if there is just a single way to access it?

Reply:
The standard library does that even where there's one method with name X, it names it X_mut https://doc.rust-lang.org/std/sync/struct.Arc.html#method.get_mut (it doesn't have get, just get_mut).  Also, it increases the clarity and a bit more consistent with nameing that we have. And you may want to add normal getters at some point as well without changing the API.

P.s. you even used `get_mut` over `get` in scheduler API as well.

Reply:
Get is different though, especially because `get` is always immutable and `get_mut` is always mutable. This method isn't called `get`, it's called `scheduler`.

Fact is that every single instance on this trait that only provides mutable access drops the _mut suffix. So I'm just following the existing style not introducing something new here.